{{cookiecutter.project_slug}}/.github/workflows/release.yaml

```diff
@@ -29,6 +33,29 @@ jobs:
       - name: Install deps
         run: poetry install
 
+      - name: Run semantic release (for tests)
+      if: ${{ github.event_name != 'workflow_dispatch' }}
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: make release-dry
```

Comment:
the project is not using make anymore, instead use makim
PS: also replace make by makim in the other steps as well. thanks!

Reply:
I have doubts here, the general project has the .makim.yaml file and in the template is the makefile and not the makim file.

Reply:
oh you are right .. sorry I didn't realized that. it would be nice to move it to makim. but it would be a separated PR.
let me check again the error.